docs/resources.rst

```diff
@@ -0,0 +1,81 @@
+Resources
+=========
+
+Tutorials
+---------
+
+Django Debugging Tutorial
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Originally presented as an in-person workshop at DjangoCon US 2022, this
+tutorial by **Tim Schilling** covers debugging techniques in Django. Follow
+along independently using the slides and GitHub repository.
+
+* `Read the blog post <https://www.better-simple.com/django/2022/11/18/django-debugging-tutorial/>`__
+* `Follow along with the GitHub repo <https://github.com/tim-schilling/debug-tutorial/>`__
+* Last updated: February 13, 2025.
+* Estimated time to complete: 1-2 hours.
+
+Mastering Django Debug Toolbar: Efficient Debugging and Optimization Techniques
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This tutorial by **Bob Berderbos** provides an in-depth look at effectively
+using Django Debug Toolbar to debug Django applications, covering installation,
+configuration, and practical usage.
+
+* `Watch on YouTube <https://www.youtube.com/watch?v=c5riXBYFxLk>`__
+* Published: May 13, 2023.
+* Duration: 11 minutes.
+
+Talks
+-----
+
+A Related Matter: Optimizing Your Web App by Using Django Debug Toolbar
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presented at DjangoCon US 2024 by **Christopher Adams**, this talk delves into
+optimizing web applications using Django Debug Toolbar, focusing on SQL query
+analysis and performance improvements.
+
+* `Watch on DjangoTV <https://djangotv.com/videos/djangocon-us/2024/a-related-matter-optimizing-your-webapp-by-using-django-debug-toolbar-with-christopher-adams/>`__
+* `Watch on YouTube <https://www.youtube.com/watch?v=iKn5VWD9l6E>`__
```

Comment:
```suggestion
* `Watch on YouTube <https://www.youtube.com/watch?v=iKn5VWD9l6E>`__
* `View the tutorial details <https://2024.djangocon.us/talks/a-related-matter-optimizing-your-webapp-by-using-django-debug-toolbar-select-related-and-prefetch-related/>`__
```

I think we should include the links to the conference sites as well (or in place of YouTube) to help route traffic to them.